chia/wallet/puzzles/pool_escaping_innerpuz.clvm

```diff
@@ -0,0 +1,31 @@
+(mod (POOL_PUZHASH RELATIVE_LOCK_HEIGHT OWNER_PUBKEY P2_SINGLETON_PUZHASH spend_type my_puzhash my_amount pool_reward_amount pool_reward_height)
+
+  ; POOL_PUZHASH is commitment to the pool's puzzlehash
+  ; RELATIVE_LOCK_HEIGHT is how long it takes to leave
+  ; ESCAPE_MODE_PUZHASH is the puzzle_hash you'll go to when you iniate the leaving process
+  ; P2_SINGLETON_PUZHASH is the puzzlehash for your pay_to_singleton puzzle
+
+  ; spend_type is: 0 for absorbing money, 1 to escape
+
+  ; absorb_coin_id and my_puzhash are only used for absorbtion case
+
+  (include condition_codes.clvm)
+  (defconstant GENESIS_CHALLENGE 0xccd5bb71183532bff220ba46c268991a)
```

Comment:
This is not the genesis challenge, it's only the first half

Reply:
Yeah, maybe call it `POOL_REWARD_PARENT_PREFIX` or something